services/calculate_analytics/analyses/calculate_feed_topic_models_2025_08_18/train/submit_topic_modeling_training.sh

```
@@ -0,0 +1,72 @@
+            ;;
+    esac
+done
+
+echo "Running topic model training with:"
+echo "  Mode: $MODE"
+echo "  Sample per day: $SAMPLE_PER_DAY"
+echo "  Output dir: ${OUTPUT_DIR:-'default'}"
+echo "  Force fallback: ${FORCE_FALLBACK:-'no'}"
+
+# Run the topic modeling training
+source $CONDA_PATH && conda activate bluesky_research && export PYTHONPATH=$PYTHONPATH
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)